mission-control-client/src/styles/components/admin-styles/admin-dashboard-content.scss

```diff
@@ -49,11 +45,11 @@
       }
 
       .dashboard-product-status {
-        padding: 8px 31px;
+        font-size: 16px;
```

Comment:
Should we do font-size in rems?

Reply:
Rems if possible. The CSS reset has the base font size at 10px so rems are easy to calculate

Reply:
Ok, I agree